### PR TITLE
Focus search field on page load

### PR DESCRIPTION
**Pull request by haplo**
@lorenzogil please take a look.

### CODE REVIEW

File: app/templates/secrets.hbs [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <span class="icon icon-menu">menu</span>
       {{/link-to}}
       <form action="#">
-        {{input placeholder="Search..." value=query}}
+        {{input id="search" placeholder="Search..." value=query}}
```
Comment (lorenzogil) [inline app/templates/secrets.hbs]: The {{ input }} ember component already defines an id, which you are overriding. I'm not sure if this a good idea.

Comment (haplo) [inline app/templates/secrets.hbs]: OK, I will take a look at the default id and try to use that.

Comment (lorenzogil) [inline app/templates/secrets.hbs]: That id is dynamic. I suggest you to take a look at how I do it at the 'secret-revealer' view. Hint: use a different selector, not the id.

Comment (haplo) [inline app/templates/secrets.hbs]: According to [this Ember documentation](http://guides.emberjs.com/v1.10.0/views/customizing-a-views-element/) it seems to be OK to override the `id` parameter of elements of the templates, it looks like Ember will use it.

**Pull request merged**